test/components/LaunchDarkly.test.js

```diff
@@ -1,36 +1,24 @@
 import React from "react";
-import { shallow } from "enzyme";
-import { Broadcast } from "react-broadcast";
+import { configure, shallow } from "enzyme";
 import LaunchDarkly from "../../src/components/LaunchDarkly";
-import { BROADCAST_CHANNEL } from "../../src/constants/LaunchDarkly";
 
-describe("components/LaunchDarkly", () => {
-  it("sets up a broadcast with the correct channel", () => {
-    const subject = shallow(
-      <LaunchDarkly clientId="080808" user="zeke">
-        <div>Hi</div>
-      </LaunchDarkly>
-    );
-
-    const broadcast = subject.find(Broadcast);
-    expect(broadcast.prop("channel")).toEqual(BROADCAST_CHANNEL);
-  });
+const Adapter = require("enzyme-adapter-react-16");
+configure({ adapter: new Adapter() });
 
+describe("components/LaunchDarkly", () => {
   it("broadcasts the config object", () => {
```

Comment:
The name of the test could probably use an update as well: `it("renders the context provider", () => {...`, or something like that.